src/app.js

```diff
@@ -1 +1,53 @@
-'use strict';
+import readline from 'readline';
+
+const args = process.argv.slice(2);
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+switch (args[0]) {
+  case 'add':
+    const resAdd = args.slice(1).reduce((acc, item) => {
```

Comment:
Try to give clear names for your variables, so that your code will be more clean and understandable
Please fix all cases